core/src/components/item/item.md.scss

```diff
@@ -83,6 +86,20 @@
   --show-inset-highlight: 0;
 }
 
+:host(.item-fill-outline) .item-highlight {
```

Comment:
I think we could simplify this down the road, but for now let's just add some comments that explain why this code is necessary and what it does.